api/src/database/migrations/20220429C-drop-non-null-from-ip-and-user-agent-in-activity.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_activity', (table) => {
+		table.setNullable('ip');
+		table.setNullable('user_agent');
```

Comment:
As this is breaking CI for Oracle, I believe it's due to `user_agent` was unintentionally nullable in the seed file, so Oracle throws this `ORA-01451: column to be modified to NULL cannot be modified to NULL` [for column that already allows null](https://docs.oracle.com/cd/A58617_01/server.804/a58312/newch223.htm#:~:text=ORA%2D01451%3A%20column%20to%20be,unnecessarily%2C%20from%20NULL%20to%20NULL.).

To share more detail on the "unintentionally nullable" part, coincidentally this was brought up recently over at https://github.com/directus/directus/pull/13072#discussion_r862670719, where there is indeed a typo `nullabel` instead of `nullable`:

https://github.com/directus/directus/blob/fa35a6b8dd3f186fdfa2f376ded824d413f8f1f7/api/src/database/seeds/05-activity.yaml#L20-L23

and I can confirm on an existing Postgres database (with migrations on main branch) does show it as nullable:

![image](https://user-images.githubusercontent.com/42867097/169433540-9a55d55c-6a38-42dd-bc34-596bd154efd0.png)

So perhaps omitting this change to `user_agent` will fix the CI. But I do wonder should we (somehow) document this knowledge formally. Maybe keeping this line as a comment with a brief explanation? 🤔

Reply:
Good find! I'm wondering if in this specific case it would be fine to remove `nullabel: false` from the seed to reduce confusion. With the spelling error, this is a noop anyways and we need the `user_agent` to be nullable for activity logs of schedule flows to work (cc @rijkvanzanten).

Reply:
We can remove the typo nullable from that file 👍🏻 

Also, there should be a migrations helper that solves the Oracle nullable problem. It's already used in some other migrations as well